pkg/generator/ingress_translator.go

```diff
@@ -35,7 +37,7 @@ import (
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/types"
 	envoy "knative.dev/net-kourier/pkg/envoy/api"
-	"knative.dev/net-kourier/pkg/reconciler/ingress/config"
+	IngressConfig "knative.dev/net-kourier/pkg/reconciler/ingress/config"
```

Comment:
Let's try to avoid unnecessary change.
```suggestion
	"knative.dev/net-kourier/pkg/reconciler/ingress/config"
```